boofuzz/primitives/fuzz_list.py

```diff
@@ -0,0 +1,125 @@
+import random
+import glob
+
+from .base_primitive import BasePrimitive
+
+
+class FuzzList(BasePrimitive):
```

Comment:
I prefer use max_len to use only anomalies with maximum length minor or equals to max_len.
For now I don't like truncate or add padding to anomalies generated by radamsa (I made a mistake in the code before your review), I think that modifies its output make it less useful. I hope I made clear my idea.